src/viam/ur/module/ur_arm.cpp

```diff
@@ -529,7 +529,7 @@ void URArm::resume_trajectory_(std::shared_lock<std::shared_mutex> config_rlock)
     }
     auto failed_trajectory = current_state_->get_failed_trajectory();
 
-    if (failed_trajectory.size() == 0) {
+    if (failed_trajectory.empty()) {
```

Comment:
[q]: what is the difference between these two?

Reply:
the linter yells at me if I do the first one